pkg/client/cluster.go

```diff
@@ -545,6 +545,8 @@ ClusterCreatOpts:
 		}
 
 		cluster.ServerLoadBalancer.Node.HookActions = append(cluster.ServerLoadBalancer.Node.HookActions, writeLbConfigAction)
+		cluster.ServerLoadBalancer.Node.RuntimeLabels = clusterCreateOpts.GlobalLabels
```

Comment:
I think this is not needed, as you already pass them to `LoadBalancerPrepare` and you would override potential other labels.
```suggestion
```